docs/softlayer_virtual_guest.md

```diff
@@ -78,8 +78,8 @@ The following arguments are supported:
 * `os_reference_code` | *string*
     * An operating system reference code that will be used to provision the computing instance.
     * **Conflicts with** `image_id`.
-* `image_id` | *string*
-    * A global identifier for the image template to be used to provision the computing instance.
+* `image_id` | *int*
+    * The image template id to be used to provision the computing instance. Note this is not the global identifier (uuid), but the image template group id that should point to a valid global identifier.
     * **Conflicts with** `os_reference_code`.
```

Comment:
@minsikl, @riethm, I like _getVhdImportSoftwareDescriptions_ better because you can mask it, in contrast with VM.getCreateObjectOptions.

```go
service := services.GetVirtualGuestBlockDeviceTemplateGroupService(sess)
descs, err := service.Mask("referenceCode").GetVhdImportSoftwareDescriptions()
```